src/supervisor/utility.py

```diff
@@ -13,6 +13,21 @@
 masterDF = pandas.DataFrame()
 nodeDetails={}
 
+acmNamespace = "open-cluster-management"
+mceNamespace = "multicluster-engine"
+
+def getACMNamespace() :
+    global acmNamespace
+    return acmNamespace
+
+def getACMNamespacesExpression() :
+    global acmNamespace, mceNamespace
+    return mceNamespace+"|"+acmNamespace+"|open-cluster-managemen.+"
+
+def getACMNamespacesExpressionNoObs() :
+    global acmNamespace, mceNamespace
+    return mceNamespace+"|"+acmNamespace+"|open-cluster-management-agent.+|open-cluster-management-hub|open-cluster-management-addon.+"
+
```

Comment:
Should we also add a `obsNamespace=open-cluster-management-observability `here

Reply:
I was assuming most of the namespaces can't change in this PR and was really just addressing the case where multicluster-engine or open-cluster-management could be different.  If open-cluster-management-observability can be customized I'd be happy to add it, otherwise it might be best if it's part of some additional namespace refactoring that I think could be done.